### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59133]
#### VFS team name
Modernized Check-in

#### VFS product name
Patient Check-in

#### VFS feature name
Beneficiary Travel

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
After submitting the travel reimbursement and checking in, there is a page with 2 stacked alerts. Per [our guidance](https://design.va.gov/components/alert#how-to-use-alerts) you cannot stack alerts.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/43074-Allison-040701.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
One suggestion is to move the green confirmation alert directly after submitting the travel claim.

If technical constraints require the green confirmation alert to appear only after you've checked into an appointment, consider moving the alert just after the "You're checked in" H1. This ensures that the feedback on both of the actions you've taken (checked in and submitted a travel claim) are the first two items announced for screen reader users ("You're checked in. We're processing your travel claim.").

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by benbrasso-agile6** [comment]
Created ticket https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/59165

**Comment by shiragoodman** [comment]
@benbrasso-agile6 as stated above in the VFS Guidance, please do not close the ticket until the issue has been resolved. Thank you for your understanding.

**Comment by loripusey** [comment]
Released on 1/22; closing
